product_docs/docs/biganimal/release/using_cluster/managing_cloud_account/csp_tagging.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: "Tagging Cloud Service Providers"
+deepToC: true
+---
+
+Tags are key-value pairs you can apply to resources created on Your Cloud Account for fine-grained access control and granular cost data at the bucket level.
+
+Tags are created at the project level and managed through the Cloud Service, which is used across your cloud region.
+
+## Create Tags
+
+To create a tag:
+
+1.  Log in to the Console.
+    
+1.  Go to the project's page.
+
+1.  In the project's list, select your **Project**.
+
+1.  On your project's home page left navigation, select **Cloud Providers**.
+
+1.  On the cloud provider's page, select **Your Cloud Account**.
+
+1.  On your cloud account tab, under **AWS**, select **Manage Tags**.
+
+1.  Select **Add a Key Value pair**.
+
+1.  Provide **Key** and **Value** and select **Save**.
+
+1.  A **Confirm changes to AWS tags** message pops up.
+
+1.  Select **Confirm** to add the tag.
+
+1.  This **Key:Value** pair is added at your project level. It is inherited by all the resources in your project including regions and clusters.
+
+This new tag appears in the list under **AWS** on Cloud Service Providers. 
+
+This tag is also added to the tags on your AWS Console. You can view this tag in the **Tags** tab of your EC2 instance resources on the AWS console.
+
+### Considerations
+
+Consider following while adding a **Key:Value** pair:
+
+- You can add a **Key:Value** pair or only **Key** in the **Key** field to add a tag.
+
+- You cannot leave the **Key** field empty.
+
+- The **Key** value must be unique within your project.
+
+- The **Key** cannot have **aws** or **managedby** in the prefix.
+
+- The maximum length for a **Key** is 127 characters and for **Value**, 255 characters.
+
+- A resource can have a maximum number of 50 tags. The number of tags per resource is defined by the tags created in the EDB Console, in your cloud service provider's dashboard, with terraform, and other synced systems.
+
```

Comment:
@amine-khemissi I reworked this section, let me know what you think. I'd prefer keeping the word considerations instead of limitations, as I don't expect many customers encountering these issues, and limitations has a negative connotation, but I will bring this up with the docs team. 

@navnathedb let me know if there are other BA reserved word we should add here.